tests/events/EventCard/eventUI.test.tsx

```diff
@@ -0,0 +1,161 @@
+            new Date("2023-02-26T19:00:00"),
+            true
+          )}
+        />
+      </MenuProvider>
+    )
+
+    fireEvent.press(screen.getByTestId(moreButtonLabel))
+    expect(screen.queryByText("Delete")).toBeDefined()
+    expect(screen.queryByText("Report")).toBeNull()
+  })
+
+  it("Event menu displays report option when user isn't the host", () => {
+    render(
+      <MenuProvider>
+        <EventItem
+          event={TestEventItems.mockEvent(
+            new Date("2023-02-26T17:00:00"),
+            new Date("2023-02-26T19:00:00"),
+            false
+          )}
+        />
+      </MenuProvider>
+    )
+
+    fireEvent.press(screen.getByTestId(moreButtonLabel))
+    expect(screen.queryByText("Report")).toBeDefined()
+    expect(screen.queryByText("Delete")).toBeNull()
+  })
+
+  it("Clicking delete opens alert", () => {
+    render(
+      <MenuProvider>
+        <EventItem
+          event={TestEventItems.mockEvent(
+            new Date("2023-02-26T17:00:00"),
+            new Date("2023-02-26T19:00:00"),
+            true
+          )}
+        />
+      </MenuProvider>
+    )
+    fireEvent.press(screen.getByTestId(moreButtonLabel))
+    fireEvent.press(screen.getByText("Delete"))
+
+    expect(alertPresentationSpy).toHaveBeenCalled()
+  })
+
+  //* * Right now event isn't removed */
+  it("Clicking delete on alert closes alert and removes event", () => {
+    render(
+      <MenuProvider>
+        <EventItem
+          event={TestEventItems.mockEvent(
+            new Date("2023-02-26T17:00:00"),
+            new Date("2023-02-26T19:00:00"),
+            true
+          )}
+        />
+      </MenuProvider>
+    )
+    fireEvent.press(screen.getByTestId(moreButtonLabel))
+    fireEvent.press(screen.getByText("Delete"))
+
+    dismissConfirmDelete()
+    expect(screen.queryByText("Delete this event?")).toBeNull()
+  })
+
+  it("Clicking report opens alert", () => {
+    render(
+      <MenuProvider>
+        <EventItem
+          event={TestEventItems.mockEvent(
+            new Date("2023-02-26T17:00:00"),
+            new Date("2023-02-26T19:00:00"),
+            false
+          )}
+        />
+      </MenuProvider>
+    )
+    fireEvent.press(screen.getByTestId(moreButtonLabel))
+    fireEvent.press(screen.getByText("Report"))
+
+    expect(alertPresentationSpy).toHaveBeenCalled()
+  })
+
+  //* * Right now reporting isn't implemented */
+  it("Clicking report on alert closes alert and reports", () => {
+    render(
+      <MenuProvider>
+        <EventItem
+          event={TestEventItems.mockEvent(
+            new Date("2023-02-26T17:00:00"),
+            new Date("2023-02-26T19:00:00"),
+            false
+          )}
+        />
+      </MenuProvider>
+    )
+    fireEvent.press(screen.getByTestId(moreButtonLabel))
+    fireEvent.press(screen.getByText("Report"))
+
+    dismissConfirmReport()
+    expect(screen.queryByText("Report this event?")).toBeNull()
+  })
+})
```

Comment:
Tests are thorough ✅

I'll approve once we separate out the menu stuff into a reusable component and move these tests as such.